src/razor/razor.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import * as Razor from 'microsoft.aspnetcore.razor.vscode';
+import { EventStream } from '../EventStream';
+
+export async function activateRazorExtension(context: vscode.ExtensionContext, eventStream: EventStream) {
+    const razorConfig = vscode.workspace.getConfiguration('razor');
+    const configuredLanguageServerDir = razorConfig.get<string>('languageServer.directory', null);
+    const languageServerDir = configuredLanguageServerDir || path.join(context.extensionPath, '.razor');
```

Comment:
should we pass the extensionPath here instead of using context.extensionPath ?

Reply:
If you think that's better, it's fine with me. Done.